src/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -435,60 +362,8 @@ void LayerTestsCommon::Infer() {
     inferRequest.Infer();
 }
 
-void LayerTestsCommon::ConvertRefsParams() {
-    ngraph::pass::ConvertPrecision<ngraph::element::Type_t::f16, ngraph::element::Type_t::f32>().run_on_model(functionRefs);
-    ngraph::pass::ConvertPrecision<ngraph::element::Type_t::bf16, ngraph::element::Type_t::f32>().run_on_model(functionRefs);
-}
-
 std::vector<std::pair<ngraph::element::Type, std::vector<std::uint8_t>>> LayerTestsCommon::CalculateRefs() {
-    ConvertRefsParams();
-    functionRefs->validate_nodes_and_infer_types();
-
-    auto referenceInputs = std::vector<std::vector<uint8_t>>(inputs.size());
-    auto refInputsTypes = std::vector<ngraph::element::Type>(inputs.size());
-    for (std::size_t i = 0; i < inputs.size(); ++i) {
-        const auto &input = inputs[i];
-        const auto inputSize = input->byteSize();
-
-        auto &referenceInput = referenceInputs[i];
-        referenceInput.resize(inputSize);
-
-        auto memory = InferenceEngine::as<InferenceEngine::MemoryBlob>(input);
-        IE_ASSERT(memory);
-        const auto lockedMemory = memory->wmap();
-        const auto buffer = lockedMemory.as<const std::uint8_t *>();
-        std::copy(buffer, buffer + inputSize, referenceInput.data());
-
-        refInputsTypes[i] = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(memory->getTensorDesc().getPrecision());
-    }
-
-    const auto &&outputsInfo = executableNetwork.GetOutputsInfo();
-    std::vector<ngraph::element::Type_t> convertType;
-    convertType.reserve(outputsInfo.size());
-    for (const auto &output : outputsInfo) {
-        convertType.push_back(
-            FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(
-                output.second->getTensorDesc().getPrecision()));
-    }
-
-    std::vector<std::pair<ngraph::element::Type, std::vector<std::uint8_t>>> expectedOutputs;
-    switch (refMode) {
-        case INTERPRETER: {
-            expectedOutputs = ngraph::helpers::interpreterFunction(functionRefs, referenceInputs, refInputsTypes);
-            break;
-        }
-        case CONSTANT_FOLDING: {
-            const auto &foldedFunc = ngraph::helpers::foldFunction(functionRefs, referenceInputs, refInputsTypes);
-            expectedOutputs = getConstData(foldedFunc);
-            break;
-        }
-        case IE: {
-            // reference inference on device with other options and nGraph function has to be implemented here
-            break;
-        }
-    }
-
-    return expectedOutputs;
+    return {};
```

Comment:
Is this function required it just return default created object?

Reply:
Yes, we have a lot of tests that calls `CalculateRefs()` manually or inherit from it. So the goal of PR was to disable comparison, and it causes other stuff cleaning (e.g. model compilation can't be deleted just right now because some tests creates inference requests manually)